eventuate-tram-cdc-connector/src/main/java/io/eventuate/tram/cdc/connector/BinlogEntryToMessageConverter.java

```diff
@@ -10,8 +10,8 @@
   @Override
   public MessageWithDestination convert(BinlogEntry binlogEntry) {
     return new MessageWithDestination(binlogEntry.getStringColumn("destination"),
-            binlogEntry.getStringColumn("payload"),
-            JSonMapper.fromJson(binlogEntry.getStringColumn("headers"), Map.class),
+            binlogEntry.getJsonColumn("payload"),
+            JSonMapper.fromJson(binlogEntry.getJsonColumn("headers"), Map.class),
```

Comment:
It's not necessarily a JSON column. e.g. new CDC working with a customer's existing database.

Reply:
yes, method handles it, I will rename it to "getStringOrJsonColumn"